data-serving/data-service/src/util/search.ts

```diff
@@ -31,6 +31,10 @@ const keywords = new Map<string, string>([
 
 export default function parseSearchQuery(q: string): ParsedSearch {
     q = q.trim();
+    // parse() doesn't handle most-likely mistyped queries like
+    // "curator: foo@bar.com" (with a space after the semicolon).
+    // Change the query here to account for that.
```

Comment:
Nit: could you write in the comment that this is taking out the whitespace between the : and the search query? So readers don't have to parse the regex :)

Reply:
done